RCTMapboxGL/RCTMapboxGL.h

```diff
@@ -17,7 +17,7 @@
 - (instancetype)initWithEventDispatcher:(RCTEventDispatcher *)eventDispatcher;
 
 - (void)setAccessToken:(NSString *)accessToken;
-- (void)setAnnotations:(NSArray *)annotations;
+- (void)setAnnotations:(NSMutableArray *)annotations;
```

Comment:
This change is unnecessary, and you didn’t pair it with a change in the .m anyways.